scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -99,6 +118,61 @@ package object hdfs {
         .map(_.getKey.datum())
     }
 
+    /**
+     * Hadoop source flavoured distributed cache - any source supported by Hadoop Filesystem
+     * abstraction can be used as a source for distributed cache.
+     *
+     * @param path to the artifact - it can be local/HDFS/S3/GCS ...
```

Comment:
Technically local/S3/GCS can be supported if Hadoop conf is set properly? I'd prefer to not mention them to avoid confusion.